.cirrus.yml

```diff
@@ -72,6 +72,7 @@ task:
     - if [ "$PGVERSION" = 17 ]; then pkgs="$pkgs postgresql-common=$(apt-cache show postgresql-common | grep "^Version: .*pgdg" | awk '{print $2}')"; fi
     - if [ "$PGVERSION" = 17 ]; then pkgs="$pkgs postgresql-client-common=$(apt-cache show postgresql-client-common | grep "^Version: .*pgdg" | awk '{print $2}')"; fi
     - apt-get -y install $pkgs
+    - apt-get -y install libldap-dev slapd ldap-utils
```

Comment:
One alternative to using the debian packages could be to use [glauth](https://github.com/glauth/glauth/releases). GLauth is a single binary LDAP server meant for CI/dev purposes. IT also has a windows release. Could use it to make a smaller footprint while making easier compatability with windows and rocky9.

Reply:
I am not quite familiar with multiple platform. If you have a solution, feel free to push your modifications to my PR.